openlibrary/templates/lib/search_foot.html

```diff
@@ -56,11 +56,15 @@
                 <div class="searchPlacement">
                     <div class="searchFilter">
                         <div class="searchOpener">
-                            <a href="javascript:;" id="searchFoot">$_("More search options")</a>
+                            <a href="javascript:;" id="searchFoot">$_("Advanced")</a>
                         </div>
-                        <div class="searchCheckbox">
-                            <input name="has_fulltext" type="checkbox" value="true" id="ftokensbtm" />
-                            <label for="ftokensbtm">$_("Show only eBooks")</label>
+                        <div class="searchCheckboxes">
+                            <ul>
+                                <li><input name="has_fulltext" type="checkbox" value="true" id="ftokenstop" />
```

Comment:
Side note: From the days of CSS 1, hyphens were best practice (q.v. https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Underscores_in_class_and_ID_Names), but this is not so true now, though I found one single case favoring hyphens that is built into CSS 2: `[att|=val]` meaning the value is either "val" or "val-"(!).

In any case, this usage in the footer mirrors what's in the header and will in any case be revamped when we go mobile.